harness/scripts/destroy.sh.twig

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+function destroy() {
+  run docker-compose down --rmi local --volumes --remove-orphans
+
+  if [ "{{ @('docker.experimental.multiplatform_build.enabled') }}" == "yes" ]; then
```

Comment:
only if CI or local needs multi platform

Reply:
also since the time between a local env `ws install/enable` and `ws destroy` another project changes the current context, the wrong context will be wiped.

If it's necessary to support local multi-platform, perhaps name the context on `docker buildx create --name ...` so it can delete the correct one.

Otherwise if it's not necessary for local multi-platform, I'd suggest avoiding `docker buildx create` and just using the docker-local context that is default

Reply:
One thing to keep in mind for the gitlab project, there are concurrent builds on the same runner, so remembering context from command to command is required, so `docker buildx bake --builder "$CONTEXT"` rather than --use.

Reply:
I can use a named builder for sure, and restrict this to only in pipeline mode. However, with the latter, it makes it more difficult to test locally. 

Probably fine with a named builder, the only issue is tracking it across ws commands given they are each executed in their own shell I think? 

I am not happy putting this stuff in the Jenkinsfile either.

Reply:
the Jenkinsfile way is the only way that means you don't need to name the context, as it's the only way to capture env vars across stages.

There's no need to over-complicate it, if a dev needs to test locally, just give instructions to run `docker buildx create --use`, which docker buildx will report is required anyway

Reply:
ok nvm, as long as it only happens on the MY127WS_ENV=pipeline, then that's fine in bash. Please use `docker buildx bake --builder ...` though so it's portable onto concurrent build runners.

Reply:
as for the name, it can be some form of workspace.name + app.version, so its deterministic across stages.

Reply:
edited: app.version -> namespace as that's what we use for compose project name

Reply:
ah great idea :)

Reply:
also, if we use a named builder, I don't think we need to worry about whether it's local or in pipeline, as it would work for both

Reply:
> 

I still don't want that on local unless I actually needed multi-platform images for local dev, I'd prefer to re-use the cache than to wipe it every build.

That said, I'm not sure what moving the context creation to build-prod is solving, as it would only mean that `ws build` would be definitely using a separate build cache, rather than sharing it (if not a bug there). However while it's in build-prod, it doesn't affect docker-based local dev anyway.

Reply:
hmm either way `ws destroy` does the wiping, so I guess it doesn't matter too much if docker local dev uses it. That said I think with buildkit (docker desktop default) we don't really need to wipe the builder cache if it's size limited and only one cache.

Reply:
yeah you are right actually, I will restrict this to pipeline only and make a note in the docs

Reply:
actually, I'll do it in this PR as this is not in a working-state for projects yet, e.g.

```
error: invalid name 2aabc0ff945a4b889d3ce603bf7d72608eebf710-2aabc0ff945a4b889d3ce603bf7d72608eebf710, name needs to start with a letter and may not contain symbols, except ._-
```
because it does not start with a letter